editor/plugins/canvas_item_editor_plugin.cpp

```diff
@@ -4385,44 +4385,19 @@ void CanvasItemEditor::_update_scrollbars() {
 	Size2 size = viewport->get_size();
 	Point2 begin = canvas_item_rect.position;
 	Point2 end = canvas_item_rect.position + canvas_item_rect.size - local_rect.size / zoom;
```

Comment:
`begin` and `end` should be removed too. Also comment needs update.

Reply:
`begin` and `end` are still used below, I can't compile the code if I remove them.

Reply:
Ah right .-.
But the comment in line 4386 mentions constrain, so it should be changed I guess.